Assignment/solution_student/css/style.css

```diff
@@ -27,15 +27,179 @@ html {
 .clearfix:after {
   clear: both;
 }
-/******************************************
-/* BASE STYLES
-/*******************************************/
+
 body {
-  color: black;
-  font-size: 16px;
+  font-size: 13px;
+  font-weight: lighter;
   background: white;
+  padding: 50px;
+  color: #05080C;
+  line-height: 1.5;
+  max-width: 900px;
+  margin: 70px auto;
+  background: #F6F6F6;
+  border-top: 5px solid #222222;
+  font-family: 'open sans',sans-serif;
+  border-bottom: 5px solid #222222;
+  padding-top: none;
+}
+
+header {
+    height: 100px;
+    margin:0 auto;
+    padding:5px;
+    background:#003366;
+    text-align: left center;
+}
+
+.logo {
+	color: white;
+	text-indent: 5px;
+	display: inline;
+}
+
+.nav li{
+	text-align: right center;
+	display: inline-block;
+	float: right;
+	padding: 0;
+	margin: 0;
+	margin-bottom: 20px;
+	list-style: none;
+	width:auto;	
+	color:#f9e42e;
+	font-size: 15px;
+	font-weight: 300px;
+}
+
+.readmore {
+	font-size: 15px;
+	font-weight: bold;
+	float: right;
+	padding-top: 5px;
+	text-decoration: none;
+	margin-bottom: 15px;
+}
+
+.implemented, .traveled {
+	width: 70%;
+	padding: 10px;
+	margin-bottom: 0;
+	height: auto;
+	border-bottom: solid black 1px;
+	display: inline-block;
+	clear: both;
+	overflow: hidden;
+	float: left;
+}
+
+.traveled{
+	margin-bottom: 100px;
+}
+
+
+
+h2, h3 { 
+	font-size: 28px;
+	font-weight: normal;
+}
+
+p::first-letter {
+	font-size: xx-large;
+	font-weight: bold;
+	float: left;
+	margin: 0 2px 2px 0;
+	border: 0;
+}
+
+.column2 {
+	float: right;
+	width: 20%;
+	clear: right;
+	margin-top: -650px;
 }
 
+.side {
+	padding: 10 15 11 5;
```

Comment:
you forgot the units! 10 what? Pizzas? haha — remember to put your units if you are using a number value besides 0!